lib/resourcebuilder/apps.go

```diff
@@ -58,7 +58,7 @@ func (b *deploymentBuilder) Do(ctx context.Context) error {
 		if err != nil && !errors.IsNotFound(err) {
 			return err
 		}
-		err = updatePodSpecWithProxy(
+		err = upgradePodSpecWithProxy(
```

Comment:
We do not need to change this or similar code. We should only focus on documentation changes and if anyone complains we can change the CLI options.